src/components/ShareTrajectoryModal/EmbedSnippetPanel.tsx

```diff
@@ -36,7 +36,7 @@ const EmbedSnippetPanel = ({ startTime }: EmbedSnippetPanelProps) => {
     const [showPreview, setShowPreview] = React.useState(false);
 
     const url = `https://simularium.allencell.org/embed?trajFileName=${trajectory}&t=${startTime}`;
```

Comment:
```suggestion
    const url = `http://${location.host}/embed?trajFileName=${trajectory}&t=${startTime}}`;
```
Per your suggestion on the other PR, this renders the appropriate preview